svsim/src/main/scala/Workspace.scala

```diff
@@ -97,7 +97,7 @@ final class Workspace(
       l("module ", Workspace.testbenchModuleName, ";")
       for ((port, index) <- ports) {
         if (port.isSettable) {
-      l("  reg  [$bits(", dut.instanceName, ".",  port.name, ")-1:0] ", port.name, " = '0;")
+      l("  bit  [$bits(", dut.instanceName, ".",  port.name, ")-1:0] ", port.name, " = '0;")
```

Comment:
Later in this same file, the DPI functions like `setBitsImpl_*` are already using `bit` as their data type. Ignoring the hope for a VCS xprop fix, it does seem more consistent to use `bit` throughout. I can't see a reason to go from 2-state to 4-state unless the initial 4-state value is intended to be significant.

Reply:
Makes sense to me